css/style.css

```diff
@@ -1,6 +1,21 @@
+.navbar-inner {
```

Comment:
Some of this work was done a long, long time ago, so I can't make comments on individual formatting changes except put up a publicly-viewable dataset on our instance to let you have a play. Would that help?